CSSGapDecorations/explainer.md

```diff
@@ -397,65 +397,111 @@ properties. See [CSSWG Issue
 
 ### Placement of gap decorations
 
-Allow authors to specify where gap decorations start and end within a container.
+An author may want to apply different sets of gap decorations to different
+regions of a given container layout. We refer to such regions as a *gap
+decoration areas*.
 
-An author may specify more than one such region and apply a different set of gap
-decorations to each. Within this document, we refer to such a region as a *gap
-decoration area*. Much like CSS Transitions and Animations, all gap decoration
-properties may take a comma-delimited list of values. Each entry in such a list
-is applied to the corresponding entry in the list of gap decoration areas. If a
-given property's list length is shorter than the gap decoration area list
-length, the shorter list cycles back to the beginning as needed.
+The author defines these areas using the `rule-areas` property. In grid and
+masonry containers, these areas are defined using tuples of line names or numbers,
+similar to the `grid-area` property, each of which is assigned a name:
 
-Gap decoration area properties are defined per container type.
+```css
+  rule-areas: --first-row 1 / 1 / 2 / -1, --first-column 1 / 1 / -1 / 2;
```

Comment:
I'm actually somewhat unclear on how the names map here. Likely worth a description below this to explain the syntax a bit

Reply:
Added some explanatory text.